File: capilidistancetraveled.cpp
/* SECOND PROGRAMMING EXERCISE */
/* SUBMITTED BY: ANDREI N. CAPILI (BSCPE 1-1 - PUP BC) */
/* C++ ACTIVITY */ 

#include <iostream>
#include <iomanip>
using namespace std;

string greetUser (){
        string userName;
        cout<<"Hi! What's your name?:";
        cin>>userName;
        cout<<"Greetings! " + userName + ", This program will help you calculate your BMI distance traveled."<<endl;
        cout<<"All you need to do is provide the necessary information below."<<endl;
        return userName;
    };

int main(){
    string userName;
    userName = greetUser();
    int hours;
    int hourCounter = 0;
    int speed = 0;
    int unit;
    
    cout << "Type 1 for metric or 2 for english units \n";
    cin >> unit;

    if(unit == 1){
        cout << "Enter the speed of the vehicle in kilometers per hour (km/h) \n";
        cin >> speed;
    }else if (unit == 2){
        cout<< "Enter the speed of the vehicle in miles per hour (mp/h) \n";
        cin >> speed;
    }else{
        cout << "You have entered an invalid input. Please rerun the program \n";
        exit(0);
    }

    cout << "Enter the hours the vehicle has traveled \n";
    cin >> hours;

    cout << "Time Spent           Distance Traveled \n";
    cout << "_____________________________________________ \n";

    if (unit == 1){
        for(int loop = 1; loop < hours+1; loop++){
            if(loop == 1){
                hourCounter++;
                cout << hourCounter << " Hour" << "              " << (hourCounter*speed) << " Kilometers \n";
            }else{
                hourCounter++;
                cout << hourCounter << " Hours " << "            " << (hourCounter*speed) << " Kilometers \n";
            }            
        }
    }else{
        for(int loop = 1; loop < hours+1; loop++){
            if(loop == 1){
                hourCounter++;
                cout << hourCounter << " Hour" << "             " << (hourCounter*speed) << " Miles \n";
            }else{
                hourCounter++;
                cout << hourCounter << " Hours " << "            " << (hourCounter*speed) << " Miles \n";
            }            
        }
    }
}

File: capilidistancetraveled.java
// THIRD PROGRAMMING EXERCISE //
// SUBMITTED BY: ANDREI N. CAPILI (BSCPE 1-1 - PUP BC)
// DISTANCE TRAVELED (DISTANCE CALCULATOR)

package javafundamentals;

	import java.util.Scanner;
	import java.util.InputMismatchException;
	
	public class capilidistancetraveled {

	    public static void main(String[]args){

	        int hours;
	        int hourCounter = 0;
	        int speed = 0;
	        int unit;
	        Scanner input = new Scanner(System.in);
	        
	        System.out.println("Hello there!, I am designed to to calculate the distance traveled. I am pleased to be your assistant for today!");
	        ;
	        
	        Scanner input1 = new Scanner(System.in);
	        System.out.print("Hi! What's your name?: ");
	        String userName = input1.nextLine();
	        introText(userName);

	     
	        
	        System.out.println("Type 1 for metric or 2 for english units");
	        unit = input1.nextInt();

	        try{

	            if(unit == 1){
	                System.out.println("Enter the speed of the vehicle in kilometers per hour (kp/h)");
	                speed = input1.nextInt();
	            }else if(unit == 2){
	                System.out.println("Enter the speed of the vehicle in miles per hour (mp/h)");
	                speed = input1.nextInt();
	            }else{
	                System.out.println("You have entered an invalid input. Please rerun the program");
	                System.exit(0);
	            }
	          
	            System.out.println("Enter the number of hours the vehicle has traveled");
	            hours = input1.nextInt();
	            input1.close();
	    
	            System.out.println();
	    
	            
	            System.out.println("Time Spent                  Distance Traveled");
	            System.out.println("_____________________________________________");
	            
	            if(unit == 1){
	                for(int loop = 1; loop < hours+1; loop ++){       
	                    if(hourCounter == 1){
	                        hourCounter++;
	                        System.out.println(hourCounter + " Hour" + "                              " + (hourCounter*speed) + " Kilometers");    
	                    }else{
	                        hourCounter++;
	                        System.out.println(hourCounter + " Hours" + "                             " + (hourCounter*speed) + " Kilometers");
	                    }
	                    
	                }
	            }
	            else{

	                for(int loop = 1; loop < hours+1; loop ++){       
	                    if(loop == 1){
	                        hourCounter++;
	                        System.out.println(hourCounter + " Hour" + "                               " + (hourCounter*speed) + " Miles");    
	                    }else{
	                        hourCounter++;
	                        System.out.println(hourCounter + " Hours" + "                             " + (hourCounter*speed) + " Miles");
	                    }
	                    
	                }
	            }
	            
	        }catch(InputMismatchException e){
	            System.out.println("Invalid input. Please rerun the program");
	        }

	    }

		private static void introText(String userName) {
			// TODO Auto-generated method stub
			
		}
	    
	}